src/components/BaseMarkdown.vue

```diff
@@ -46,12 +59,36 @@ onMounted(() => {
       });
       code.appendChild(copyButton);
     });
+    body.querySelectorAll('a[href]').forEach(function (link) {
+      link.addEventListener('click', function (e) {
+        handleLinkClick(e, link.getAttribute('href'));
+      });
+    });
+  }
```

Comment:
I think rather than cutting it off the whole URL should be visible and line broken like this:
<img width="505" alt="image" src="https://user-images.githubusercontent.com/51686767/233395561-d90abfee-de9d-4705-b1ae-756138d8231a.png">

Reply:
This only affects the markdown body so I don't see the need to include snapshot and custom domain URL